android/src/main/kotlin/live/hms/hmssdk_flutter/HmssdkFlutterPlugin.kt

```diff
@@ -576,9 +595,33 @@ class HmssdkFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
         result.success(HMSPeerExtension.toDictionary(getLocalPeer()))
     }
 
+
+    private fun startScreenShare() {
```

Comment:
`startScreenShare` function should return `result.success(nil)` if successful or exception hashmap on error